k8s/namespaces/fees-pay/fees-register-frontend/ithc.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: fees-register-frontend
+spec:
+  releaseName: fees-register-frontend
```

Comment:
Is ithc.yaml missing?

```
      environment:
        DUMMY_RESTART_VAR: true
```

Reply:
added environment:
        DUMMY_RESTART_VAR: true